Bio/SubsMat/FreqTable.py

```diff
@@ -38,10 +38,18 @@
   count: count dictionary. Empty if no counts are provided.
 
 Example of use:
-  >>> from SubsMat import FreqTable
-  >>> ftab = FreqTable.FreqTable(my_frequency_dictionary,FreqTable.FREQ)
-  >>> ftab = FreqTable.FreqTable(my_count_dictionary,FreqTable.COUNT)
-  >>> ftab = FreqTable.read_count(open('myDNACountFile'))
+    >>> import io
+    >>> from Bio.SubsMat import FreqTable
+    >>> f_count = io.StringIO("A  50\nC  37\nG  23\nT  58")
+    >>> ftab = FreqTable.read_count(f_count)
+    >>> nucleobases = list(ftab.keys())
+    >>> nucleobases.sort()
+    >>> for nb in nucleobases:
+    ...     print("%s %0.4f" %(nb, ftab[nb]))
```

Comment:
Can you add a line:

```python
...
```

to mark the end of the for loop, as you would see doing this at the Python prompt please.